README.md

```diff
@@ -428,6 +429,141 @@ cards: [
 }
 ```
 
+# Portfolio Project Contribution Guide
+
+Welcome to the **Portfolio Project Contribution Guide**! This guide provides step-by-step instructions for adding a new portfolio project to this repository.
+
+---
+
+## Table of Contents
+
+1. [Project Setup](#setup-and-installation)
+2. [Steps for Adding a New Portfolio Project](#steps-for-adding-a-new-portfolio-project)
+   - [1. Add an id](#1-add-an-id)
+   - [2. Add the Preview Image](#2-add-a-preview-image)
+   - [3. Add Slider Images](#3-add-slider-images)
+   - [4. Add Category Links](#4-add-category-links)
+   - [5. Add Title](#5-add-title)
+   - [6. Add Category](#6-add-category)
+   - [7. Add Project Type](#7-add-project-type)
+   - [8. Add Client Name](#8-add-client-name)
+   - [9. Add Date](#9-add-date)
+   - [10. Add Brief Description](#10-add-a-brief-description)
+   - [11. Add Client's Challenge](#11-add-the-challenge-client-faced)
+   - [12. Add Final Results](#12-add-final-results)
+3. [Final Example Portfolio Object](#final-example-portfolio-object)
+
+---
+
+## Steps for Adding a New Portfolio Project
+
+You need to modify the `VAPTPortfolio` or `PDPortfolio` list in `data/portfolio.js` and add the final object to the respective list to upload a new portfolio project.
+
+### 1. Add an ID
+The `id` (required) is the unique identifier for a portfolio project. Adding duplicates can lead to unexpected issues in other projects. You need to assign an `id` that is one value greater than the previous project's `id` to maintain order. For example, if the previous project's `id` is `101`, use `102`, as shown below:
+```javascript
+id: 102,
+```
+
+### 2. Add a Preview Image
+The `imageSrc` (required) is used as the display picture for the project. Upload an image (dimensions: 1130x1420) to the respective folder based on the project type:  
+
+- For type `pd` (Product Design): `public/assets/img/portfolio/portfolio-pd`  
+- For type `vapt` (Penetration Testing & Vulnerability Assessment): `public/assets/img/portfolio/portfolio-pavt`  
```

Comment:
its vapt not pavt

Reply:
Technically, it should be `vapt`, but there's a typo in the name of the directory: `public/assets/img/portfolio/portfolio-pavt`:
![image](https://github.com/user-attachments/assets/beeaccbd-6df8-4c9a-9f74-ea9af4dfe092)

Reply:
please fix it.